src/main/java/TaskList.java

```diff
@@ -0,0 +1,176 @@
+
+
+import java.util.ArrayList;
+import java.time.LocalDateTime;
+import java.lang.StringBuilder;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.time.format.DateTimeFormatter;
+
+
+public class TaskList {
+    private ArrayList<Task> list;
```

Comment:
Although this code is just a WIP, I would just like to leave a comment: perhaps changing out the private ArrayList "list" with tasks (note plural), or a more descriptive name would be better! According to the coding conventions, it would be better for a list/arraylist to be named its elements, and in the plural form (with the s). 

> Plural form should be used on names representing a collection of objects. - (the coding document thing)

Reply:
Overall, I don't see many things (at least to me) that should be changed! I would just like to stress on the collection naming conventions. Add more javadocs and it should be looking good. Thanks so much for reading!

Reply:
Thank you for your comments!